### ISSUE TITLE
TypeError: 'DocstringReturns' object is not iterable

### ISSUE THREAD

**Issue by RansSelected** [rr-/docstring_parser#33]
Hi!

I was parsing the Numpy docstrings and apparently, the Return object does not store more than one position. However, some functions can return several values, thus in Numpy return section there will be a few positions, like this:

```
Returns
-------
ret1 : int
    Description of returned number.
ret2 : str or None
    Description of string alternative.
```

Could you please give some hints of how to fix the issue promptly?

All best regards!

**Comment by mayitbeegh** [comment]
Second this

**Comment by rr-** [comment]
Solved by @mayitbeegh in PR #43 by introducing backwards-compatible `many_returns` property. Fix is released on PyPI as 0.9.
